editor/header/tools/style.scss

```diff
@@ -75,3 +75,34 @@
 		}
 	}
 }
+
+.editor-tools__publish-button.is-saving,
+.editor-tools__publish-button.is-saving:disabled {
+	position: relative;
+
+	// These styles overrides the disabled state with the button primary styles
+	background: none !important;
```

Comment:
Can we override by exceeding specificity of those default styles, instead of through `!important` ? Why are we breaking consistency with default disabled button styling ? Should we be applying these overrides to `Button` more generally if they're warranted ?